ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.17.yaml

```diff
@@ -56,6 +56,36 @@ resources:
       cpu: 400m
       memory: 800Mi
 tests:
+- as: aws-ipi-byo-iam-profile-master-f28
```

Comment:
@yunjiang29 seems there's no `arm` keyword in this two job names, do we want to add it?

Reply:
Updated.